tests/foreman/api/test_smartproxy.py

```diff
@@ -41,9 +42,10 @@ def setUpClass(cls):
         existing smart proxy should always succeed.
         """
         super(SmartProxyMissingAttrTestCase, cls).setUpClass()
-        smart_proxies = entities.SmartProxy().search()
-        assert len(smart_proxies) > 0
```

Comment:
Same here + in case that assertion failed we will not run tests at all, so have error in the code, not failed cases